File: app/src/main/java/com/BlizzardArmory/warcraft/charactersummary/Collections.java
package com.BlizzardArmory.warcraft.charactersummary;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Collections {

    @SerializedName("href")
    @Expose
    private String href;

    public String getHref() {
        return href;
    }

    public void setHref(String href) {
        this.href = href;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/charactersummary/Collections.kt
package com.BlizzardArmory.model.warcraft.charactersummary

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * The type Collections.
 */
data class Collections(

        @SerializedName("href")
        @Expose
        var href: String

)